frontend/src/pages/integrations/databricks/authorize.tsx

```diff
@@ -0,0 +1,118 @@
+import { useState } from "react";
+import Head from "next/head";
+import Image from "next/image";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { faArrowUpRightFromSquare, faBookOpen } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { useSaveIntegrationAccessToken } from "@app/hooks/api";
+
+import { Button, Card, CardTitle, FormControl, Input } from "../../../components/v2";
+
+export default function DatabricksCreateIntegrationPage() {
+  const router = useRouter();
+  const { mutateAsync } = useSaveIntegrationAccessToken();
+
+  const [apiKey, setApiKey] = useState("");
+  const [instanceURL, setInstanceURL] = useState("");
+  const [apiKeyErrorText, setApiKeyErrorText] = useState("");
+  const [instanceURLErrorText, setInstanceURLErrorText] = useState("");
```

Comment:
Are you familiar with `react-hook-form`? If not I'd be happy to run through it with you! We use it in other locations in the app -  It makes field/form management with errors a lot simpler - no change required just more a curiosity